fields.go

```diff
@@ -22,15 +22,6 @@ func GetFieldsOf(st interface{}) ([]*Field, error) {
 func CollectFields(st interface{}, fields []*Field) ([]*Field, error) {
 	iter := NewFieldIteration(st)
 	for iter.Next() {
-		if iter.IsEmbeddedStruct() {
```

Comment:
this allowed scanned to or scanned from struct to have a nested struct - which translated flat columns
eg

```
type S struct {
	A int
	B struct {
		C int
	}
}
```

would translate to 3 columns a, b, c

we never used this feature and want a json column instead